### PR TITLE
Merge latest strings from activity-stream changeset f3e33da50e0bea2fad0a38a985c5eab1d98eb38a

### PR DESCRIPTION
**Pull request by Mardak**
Top Sites Edit form labels https://github.com/mozilla/activity-stream/pull/3965:
image

Section menu https://github.com/mozilla/activity-stream/pull/3987:
image
image

React 16 https://github.com/mozilla/activity-stream/pull/3964:
image

Pocket cards https://github.com/mozilla/activity-stream/pull/3975:
image

### CODE REVIEW

File: l10n/templates/strings.properties [hunk 1]
```diff
@@ -181,3 +186,24 @@ manual_migration_cancel_button=No Thanks
 # LOCALIZATION NOTE (manual_migration_import_button): This message is shown on a button that starts the process
 # of importing another browser’s profile profile into Firefox.
 manual_migration_import_button=Import Now
+
+# LOCALIZATION NOTE (error_fallback_default_info).  This message is shown when
+# there is an error rendering some part of the page in place the chunk of UI
```
Comment (flodolo) [inline l10n/templates/strings.properties]: English seems off in this comment (or, at least, I can't understand it at all)

File: l10n/templates/strings.properties [hunk 2]
```diff
@@ -181,3 +186,24 @@ manual_migration_cancel_button=No Thanks
...
+# that failed to render.
+error_fallback_default_info=Oops, something went wrong loading this content.
+# LOCALIZATION NOTE (error_fallback_default_refresh_suggestion).  This message is shown when
+# there is an error rendering some part of the page in place the chunk of UI
```
Comment (flodolo) [inline l10n/templates/strings.properties]: Same problem here with English in the comment.

**Review by flodolo — CHANGES_REQUESTED** [review]

File: l10n/templates/strings.properties (hunk 1 above)
Comment (Mardak) [reply l10n/templates/strings.properties]: @dmose wrote the note but is out.. so I'll take a stab ;)

```ini
# LOCALIZATION NOTE (error_fallback_default_*): This message and suggested
# action link are shown in each section of UI that fails to render
error_fallback_default_info=Oops, something went wrong loading this content. 
error_fallback_default_refresh_suggestion=Refresh page to try again.
```

Comment (Mardak) [reply l10n/templates/strings.properties]: @dmose, @flodolo does this NOTE adjustment seem reasonable?

Comment (flodolo) [reply l10n/templates/strings.properties]: Makes definitely sense to me

**Comment by Mardak** [comment]
@flodolo updated localization note and latest strings export also includes Pocket cards. When landing, I'll squash to a single commit using the latest commit's message.

**Review by flodolo — APPROVED** [review]

**Pull request merged**